spec/models/manageiq/providers/amazon/cloud_manager/refresh_parser_spec.rb

```diff
@@ -4,7 +4,10 @@
 
   context "#get_public_images" do
     it "applies filter from settings.yml" do
-      filter = {:filters => [{:name => "image-type", :values => ["machine"]}]}
+      cfg = Config::Options.new
+      cfg.name = "image-type"
+      cfg.values = ["machine"]
+      filter = {:filters => [cfg]}
```

Comment:
You can define the Config::Options inline: `filter = {:filters => [Config::Options.new(:name => "image-type", :values => ["machine"])]}`

Reply:
Thanks, Adam. Changed!